library/src/symbols/sym.rs

```diff
@@ -718,6 +718,11 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
     ],
     tack: [
         r: '⊢',
+        not: '⊬',
+        r.not: '⊬',
+        r.short: '⊦',
+        short: '⊦',
+        long: '⟝',
```

Comment:
```suggestion
        r.short: '⊦',
        r.not: '⊬',
```

Reply:
`tack.short` will still work even after these removals because `tack.r.short` is the first symbol in the list that has all specified modifiers. In the same way that `arrow` is the same as `arrow.r`. The documentation don't make this really clear, I'll try to improve that.